about.html

```diff
@@ -62,13 +62,15 @@ <h1>
         <h1>See the past Tape@3AM Global Summits</h1>
         <span class="redBarShort"></span>
         <p class="info">Take a look at our two last summits took place at Singapore and Taiwan</p>
-        <div class="pastSummit1">
-            <h3>2020</h3>
-            <p>Techscape@3AM Global Summit 2020 in Singapore</p>
-        </div>
-        <div class="pastSummit2">
-            <h3>2017</h3>
-            <p>Techscape@3AM Global Summit 2017 in Taiwan</p>
+        <div class="row">
+            <div class="pastSummit1">
+                <h3>2020</h3>
+                <p>Techscape@3AM Global Summit 2020 in Singapore</p>
+            </div>
+            <div class="pastSummit2">
+                <h3>2017</h3>
+                <p>Techscape@3AM Global Summit 2017 in Taiwan</p>
+            </div>
         </div>
     </section>
     <section class="partner">
```

Comment:
- [ ] On the desktop version of the About page, the partner's section is hidden, and the footer below it has a dark background color with white text and a logo. Kindly change the colors and hide the partner's section to match the template.
 - Your design:
 - 
<img width="1032" alt="image" src="https://user-images.githubusercontent.com/61919617/225435852-89a7e0a1-18e2-4f6e-aa5a-6f2e248d3706.png">

 - Template design:
 - 
<img width="747" alt="image" src="https://user-images.githubusercontent.com/61919617/225435913-ec288e88-95ae-4bb3-86ab-ce6faca7d6aa.png">